Dockerfile

```diff
@@ -23,7 +23,13 @@ RUN mkdir -p catkin_ws/src
 
 WORKDIR /home/catkin_ws/src/
 
-RUN git clone -b noetic-devel https://github.com/ROBOTIS-GIT/turtlebot3_simulations.git
+RUN git clone -b noetic-devel https://github.com/ROBOTIS-GIT/turtlebot3_simulations.git &&\
+    git clone https://github.com/ROBOTIS-GIT/turtlebot3_manipulation.git &&\
+    git clone https://github.com/ROBOTIS-GIT/turtlebot3_manipulation_simulations.git &&\
+    git clone https://github.com/ROBOTIS-GIT/open_manipulator_dependencies.git &&\
+    apt install -y ros-noetic-ros-control* ros-noetic-control* ros-noetic-moveit*
```

Comment:
Shoulld these be installed using dependency in our ros package?

Reply:
Did not manage to install as dependency, potentially due to *-asterix?

Reply:
It seems that there are multiple packages needed from each group. Could we identify which packages are needed?

Reply:
The three packages `ros_control` , `ros_controllers` and `moveit` seems to be sufficient. Tested with the simple gui and with rviz. Both seem to work now.